tests/operator/helper.go

```diff
@@ -45,3 +52,95 @@ func SplitCsv(csv string) CsvResult {
 	}
 	return result
 }
+
+func getAllPodsBy(namespace string) (podsInNamespace []*provider.Pod) {
+	for i := range env.AllPods {
+		pod := env.AllPods[i]
+		if pod.Namespace == namespace {
+			podsInNamespace = append(podsInNamespace, pod)
+		}
+	}
+	return podsInNamespace
+}
+
+func getAllOperatorsBy(namespace string) (operatorsInNamespace []*provider.Operator) {
+	for _, operator := range env.AllOperators {
+		if operator.Namespace == namespace {
+			operatorsInNamespace = append(operatorsInNamespace, operator)
+		}
+	}
+	return operatorsInNamespace
+}
+
+func isSingleNamespacedOperator(operator *provider.Operator) bool {
+	return len(operator.TargetNamespaces) == 1 && operator.Namespace != operator.TargetNamespaces[0]
+}
+
+func isMultiNamespacedOperator(operator *provider.Operator) bool {
+	return len(operator.TargetNamespaces) > 1 && !stringhelper.StringInSlice(operator.TargetNamespaces, operator.Namespace, false)
```

Comment:
I'm not sure the check for `len(operator.TargetNamespaces) > 1` is actually needed because `StringInSlice` will already return false for a slice that's 0 length.

Reply:
We need that, imo. We want to avoid that situation for ClusterWide operators. Does it make sense?